reqs/management/commands/import_reqs.py

```diff
@@ -219,12 +219,32 @@ def __init__(self, keywords=None):
         self.keywords = KeywordProcessor(keywords)
         self.connections = []
         self.req_ids = set()
+        self.last_id = "0.0"
+
+    def fix_excel_decimals(self, reqid):
+        """
+        Excel strips trailing zeroes, so for example what should be 1.20 is 1.2
+        in the data.
+        Since the rows are sequential (or should be!), we should be able to
+        account for this.
+        """
+        policy, requirement = reqid.split(".")
+        last_policy, last_requirement = self.last_id.split(".")
+        if policy == last_policy:
+            if last_requirement.endswith("9"):
+                sequential = str(int(last_requirement) + 1)
+                if sequential[:len(requirement)] == requirement:
+                    return "{0}.{1}".format(policy, sequential)
+        return reqid
 
     def add(self, row):
         req_key = find_key("req_ids", row)
         req_id = row[req_key]
         if req_id in ('None', ''):
             raise ValueError("Requirement without ID")
+        if "." not in req_id:
+            raise ValueError("Requirement without . separator")
```

Comment:
Should this check happen inside `fix_excel_decimals`? Just wondering if we can keep all that logic in one place.